src/Surfnet/StepupMiddleware/ApiBundle/Controller/RaSecondFactorController.php

```diff
@@ -85,12 +85,11 @@ public function exportAction(Request $request, Institution $actorInstitution)
      * @param Institution $actorInstitution
      * @return RaSecondFactorQuery
      */
-    private function buildRaSecondFactorQuery(Request $request, Institution $actorInstitution)
+    private function buildRaSecondFactorQuery(Request $request)
```

Comment:
The doc block mention of the `actorInstitution` can be removed now.

Reply:
Also the callers of this method still pass two parameters (they still pass the actor institution). Please update this to prevent confusion at a later stage.

Reply:
Good catch, removed the param